sipi/scripts/sipi.init.lua

```diff
@@ -123,32 +123,27 @@ function pre_flight(prefix, identifier, cookie)
         return 'deny'
     elseif permission_code == 1 then
         -- restricted view permission on file
-        -- either watermark or size (depends on project, should be returned by DSP-API)
-        -- currently, only size is used
-
-        local restrictedViewSize
-
-        if permission_info.restrictedViewSettings ~= nil then
-            log("pre_flight - restricted view settings - watermark: " ..
-                tostring(permission_info.restrictedViewSettings.watermark), server.loglevel.LOG_DEBUG)
-
-            if permission_info.restrictedViewSettings.size ~= nil then
-                restrictedViewSize = permission_info.restrictedViewSettings.size
-                log("pre_flight - restricted view settings - size: " .. tostring(restrictedViewSize),
-                    server.loglevel.LOG_DEBUG)
-            else
-                log("pre_flight - using default restricted view size", server.loglevel.LOG_DEBUG)
-                restrictedViewSize = config.thumb_size
+        -- watermark and/or size (set per project, managed by DSP-API)
+        local settings_from_api = permission_info.restrictedViewSettings
+
+        local settings_for_sipi = {}
+        settings_for_sipi['type'] = "restrict"
```

Comment:
Should this be `restrict` or  `restricted`?

Reply:
It is actually https://github.com/dasch-swiss/sipi/pull/404